domains/scratchempires.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "qloakongithub",
+           "email": "7469931481@stu.gcsnc.com",
+           "discord": "865563672553062431"
+        },
+    
+        "record": {
+            "CNAME": "scratch-empires-unofficial-organization.github.io"
```

Comment:
404 not found

Reply:
Sorry, forgot to change. Should be fixed.